.github/scripts/commit_and_push.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+readonly GIT_HAS_CHANGES=1 # 0 = clean, 1 = dirty
+
+USER_NAME="$1"
+USER_EMAIL="$2"
+
+git config --global user.name "$USER_NAME"
```

Comment:
Does this need to be a global git config? Will it work with a local git config as well?

Reply:
It doesn't have to be since it's just a VM I set as global. But changed it to repository level.